integration/docker/entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ $# -ne 1 ]]; then
+  echo 'expected one argument: "master", "worker", or "proxy"'
+  exit 1
+fi
+
+service=$1
+
+home=/opt/$(ls /opt | grep alluxio)
+cd ${home}
+
+for keyvaluepair in $(env | grep "ALLUXIO_"); do
```

Comment:
I can't think of another prefix which isn't a lot less natural. What do you think of using a blacklist of environment variables starting with `ALLUXIO_` which don't correspond to configuration keys?